scripts/end_to_end_tests.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+
+set -euf -o pipefail
+
+SCRIPT_DIR=$(pwd)
```

Comment:
This will get you the directory the script is _running_ in, and is redundant with `pwd` the point of the
```bash
SCRIPT_DIR=$(dirname ${0})
```
is to give us the location of the script _file_, so we have a canonical location in the `timescale-prometheus` file-tree from which we can find other files (issue being that the script could have been called from an arbitrary location, so we have no real way to get from `pwd` to our other stuff)

Reply:
on further investigation, something like
```bash
SCRIPT_DIR=$(cd $(dirname ${0}) && pwd)
```
may work better